tests/rptest/services/keycloak.py

```diff
@@ -209,12 +247,23 @@ def start_node(self,
                    node,
                    access_token_lifespan_s=DEFAULT_AT_LIFESPAN_S,
                    **kwargs):
-        self.logger.debug("Starting Keycloak service")
+
+        cfg_writer = KeycloakConfigWriter(
+            extra_cfg={
+                'hostname': self.host(node),
+                'hostname-port': self.http_port,
+                'http-port': self.http_port,
+                'log-level': self.log_level,
+            })
+        cfg_writer.write(node)
 
         node.account.ssh(f"touch {KC_LOG_FILE}", allow_fail=False)
 
+        self.logger.debug(f"Starting Keycloak service {cfg_writer.rep}")
+
         with node.account.monitor_log(KC_LOG_FILE) as monitor:
-            node.account.ssh_capture(self._start_cmd(node), allow_fail=False)
+            node.account.ssh_capture(f"nohup {self._start_cmd(node)}",
```

Comment:
Is `nohup` intended, or was that for debugging? If it was intended, when is it cleaned up?